eq-author/src/App/settings/ThemesPage.js

```diff
@@ -217,14 +261,28 @@ const ThemesPage = ({ questionnaire }) => {
                           data-test={`${shortName}-toggle`}
                         >
                           <p />
-                          <Field>
-                            <Label>eQ ID</Label>
-                          </Field>
-                          <EqIdInput
-                            eqId={eqId}
-                            questionnaireId={id}
-                            shortName={shortName}
-                          />
+                          <StyledIdContainerOuter>
+                            <StyledEqIdContainer>
+                              <Field>
+                                <Label>eQ ID</Label>
```

Comment:
```suggestion
                                <Label>EQ ID</Label>
```

Reply:
That was Sam 🤣

Reply:
This was changed after a discussion with Curtis. The intended text is "eQ ID", without the capital.